src/plugins/intel_gpu/src/kernel_selector/cl_kernels/mvn_gpu_bfyx_opt.cl

```diff
@@ -4,45 +4,58 @@
 
 #include "include/batch_headers/fetch_data.cl"
 
+#if !IS_DYNAMIC
 __attribute__((reqd_work_group_size(LWS, 1, 1)))
+#endif
 KERNEL (mvn_gpu_bfyx_opt)(
+    OPTIONAL_SHAPE_INFO_ARG
     const __global INPUT0_TYPE* input,
     __global OUTPUT_TYPE* restrict output
 #if HAS_FUSED_OPS_DECLS
     , FUSED_OPS_DECLS
 #endif
 ) {
-    const uint data_set_idx = get_global_id(1);     //in processing of which data set this WI participates?
-    const uint workers_per_data_set = LWS;          //how many WI participates in processing of one data set
-    const uint in_data_set_idx = get_global_id(0);  //this WI's id in group of items processing single data set
-    const uint data_set_size = DATA_SET_SIZE;       //how many elements are in one data set
-    const uint data_sets_count = DATA_SETS_COUNT;   //how many data sets are in the processing payload
+    const uint data_set_idx = get_global_id(1);                   // in processing of which data set this WI participates?
+    const uint workers_per_data_set = get_local_size(0);          // how many WI participates in processing of one data set
```

Comment:
Did you have a chance to check static case performance?
`workers_per_data_set` is not compile-time const variable anymore - wouldn't it cause performance regression? (There are quite a lot cases where workers_per_data_set is used for multiplication and can't be precalculated now)

Reply:
Updated existing compile-time const variables to be used to avoid impacting performance in the static kernel case.